server/packaging/rpm/SOURCES/stackable-opa-operator-server-VERSION/usr/lib/systemd/system/stackable-opa-operator-server.service

```diff
@@ -0,0 +1,13 @@
+[Unit]
+Description=Stackable OpenPolicyAgent Operator
+Before=
+After=network.target
+[Service]
+User=root
+ExecStart=/opt/stackable/opa-operator/opa-operator-server
```

Comment:
The binary here needs to match the parameter you pass to the buildrpm script in the nightly action.

Reply:
Thanks will change.